tools/restart_lipstick.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+nohup sh -c "pkill jolla-settings;systemctl --user --no-block restart lipstick.service" 1>/dev/null 2>/dev/null &
```

Comment:
It allows to avoid race condition, otherwise jolla-settings could wait for the restart process to end. And when lipstick stops blocked UI application will most probably crash.